docs/DeveloperGuide.md

```diff
@@ -122,6 +122,8 @@ The `Model`,
 The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
 * can save the address book data in json format and read it back.
+* depends on some classes in the `Model` component
+(because the `Storage` component's job it to save/retrieve objects belongs to the `Model`)
```

Comment:
```suggestion
(because the `Storage` component's job is to save/retrieve objects that belong to the `Model`)
```

Sorry, there was a typo in my initial suggestion that you followed.